src/collective/cover/content.py

```diff
@@ -148,6 +149,23 @@ def get_referenced_objects(self):
                 refs |= getObjectsFromLinks(self, links)
         return refs
 
+    def purge_deleted_tiles(self):
```

Comment:
we need to explain better why we are doing this; it can be in the `docstring` or in code comments